setup

```diff
@@ -266,6 +268,12 @@ superbbootstrap_AUR() {
 	fi
 }
 
+superbbootstrap_PIP() {
+  pip install pip --upgrade
```

Comment:
In many Linux distros system Python has no `pip` module installed so the this command should be:
```sh
python -m ensurepip
```
And even if it does have `pip` module, it's executable could not be on the $PATH, so the package install command should look like this:
```sh
python -m pip install --upgrade <package_name>
```
The `--upgrade` option tells it not to reinstall the package if it is already installed.